lampo-common/src/conf.rs

```diff
@@ -21,6 +21,7 @@ pub struct LampoConf {
     pub channels_keys: Option<String>,
     pub log_file: Option<String>,
     pub log_level: String,
+    pub alias: String,
```

Comment:
This is too simple, we should produce a better get info

Reply:
I am working on it. This is just one of a step to it.